src/Language/DifferentialDatalog/Compile.hs

```diff
@@ -624,13 +624,19 @@ collectStatics d = execState (progExprMapCtxM d (checkStaticExpr)) M.empty
 -- computed by 'collectStatics'.
 mkStatics :: DatalogProgram -> Statics -> Doc
 mkStatics d statics =
-    vcat $
-    map (\((e, t), i) ->
-            let static_name = "__STATIC_" <> pp i in
-            let ?statics = deleteStatic e t statics in
-            let e' = mkExpr d CtxTop (eTyped e t) EVal in
-            "lazy_static!{ pub static ref" <+> static_name <> ":" <+> mkType t <+> "=" <+> e' <+> "; }")
-        $ M.toList statics
+    "lazy_static! {"
```

Comment:
Unfortunately, this does not work.  I don't remember the exact error, but basically rustc failed when there were too many static variables within a single `lazy_static`.

Reply:
That's really bizarre, and rather unfortunate. I revered the change, should be good now